server/services/cryptoDonationService.js

```
@@ -0,0 +1,51 @@
+const USDC_ADDRESS = "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48";
+
+// Active tracking memory (in production this would be Redis/DB)
+export const activeDonationTracks = new Map();
+
+/**
+ * Listens for a specific transaction hash to be confirmed on the blockchain
+ * and verifies it matches the expected USDC donation amount.
+ */
+export const trackCryptoDonation = async (clubId, txHash, donorAddress, expectedAmount) => {
+  try {
+    activeDonationTracks.set(txHash, { status: 'PENDING', clubId, donorAddress });
```

ESLint

[error] Replace `'PENDING'` with `"PENDING"`

(prettier/prettier)